### PR TITLE
INDY-2097: Update pluggable request handlers

### PR DESCRIPTION
**Pull request by Toktar**

### CODE REVIEW

File: indy_node/server/request_handlers/read_req_handlers/get_revoc_reg_delta_handler.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def get_result(self, request: Request):
             encoded_revoc_reg_def = self.state.get_for_root_hash(entry_to.root_hash,
```
Comment (ashcherbakov) [inline indy_node/server/request_handlers/read_req_handlers/get_revoc_reg_delta_handler.py]: Can we use `get_from_state` here?

File: indy_node/server/request_handlers/read_req_handlers/get_revoc_reg_delta_handler.py [hunk 2]
```diff
@@ -116,7 +116,7 @@ def _get_reg_entry_by_timestamp(self, timestamp, path_to_reg_entry):
                                                                         head_hash=past_root,
                                                                         with_proof=True)
             if encoded_entry:
-                reg_entry, seq_no, last_update_time = domain.decode_state_value(encoded_entry)
+                reg_entry, seq_no, last_update_time = decode_state_value(encoded_entry)
```
Comment (ashcherbakov) [inline indy_node/server/request_handlers/read_req_handlers/get_revoc_reg_delta_handler.py]: Can we use `get_from_state`?

File: indy_node/server/request_handlers/read_req_handlers/get_revoc_reg_handler.py [hunk 3]
```diff
@@ -22,15 +21,15 @@ def get_result(self, request: Request):
...
+        path = RevocRegEntryHandler.make_state_path_for_revoc_reg_entry_accum(revoc_reg_def_id=revoc_reg_def_id)
         entry_state = StateValue()
         if past_root is not None:
             encoded_entry, proof = self._get_value_from_state(path,
                                                               head_hash=past_root,
                                                               with_proof=True)
             entry_state.proof = proof
             if encoded_entry:
-                revoc_reg_entry_accum, seq_no, last_update_time = domain.decode_state_value(encoded_entry)
+                revoc_reg_entry_accum, seq_no, last_update_time = decode_state_value(encoded_entry)
```
Comment (ashcherbakov) [inline indy_node/server/request_handlers/read_req_handlers/get_revoc_reg_handler.py]: Can we use `get_from_state`?

**Comment by Toktar** [comment]
Test this please

**Comment by Toktar** [comment]
test this please

**Pull request merged**